test/assert-test.js

```diff
@@ -1649,4 +1649,45 @@ describe("assert", function () {
                           "    actual = { foo: 1 }");
         });
     });
+
+    if (typeof Symbol === "function") {
+        describe("with symbol method names", function () {
+            function setupSymbol(symbol) {
+                this.obj = {};
+                this.obj[symbol] = function () {};
+                sinonSpy(this.obj, symbol);
+            }
+
+            beforeEach(function () {
+                this.setupSymbol = setupSymbol.bind(this);
+
+                /*eslint consistent-return: "off"*/
+                this.message = function (method) {
```

Comment:
Simplify into
```javascript 
function createExceptionMessage(method, arg) {
    try { sinonAssert[method](arg); }
    catch (e) { return e.message; }
}
```

All the uses have at most one argument so we can drop the slicing.